profiles/dilapitan.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/23071809/193412365-2a2d6fb6-6284-4390-bfb1-696f6d605067.png",
+  "issueId": 1422,
+  "name": "Dominic Lapitan",
+  "username": "dilapitan"
+}
```

Comment:
```suggestion
}

```

Reply:
Whoops, sorry I forgot about the last new line. Thanks!